dialogue-clients/src/test/java/com/palantir/dialogue/clients/DefaultDialogueDnsResolverTest.java

```diff
@@ -66,7 +70,42 @@ void malformedIpv6_noBrackets() {
         assertThat(resolve("::z")).isEmpty();
     }
 
+    @Test
+    void unknown_host() {
+        assumeThat(System.getProperty("os.name").toLowerCase().startsWith("linux"))
+                .describedAs("GAI Error Strings are only defined for Linux environments")
+                .isTrue();
+
+        TaggedMetricRegistry registry = new DefaultTaggedMetricRegistry();
+        DialogueDnsResolver resolver = new DefaultDialogueDnsResolver(registry);
+
+        String badHost = "alksdjflajsdlkfjalksjflkadjsf.com";
```

Comment:
Anyone could [pay ~$10](https://porkbun.com/checkout/search?prb=9261189adc&q=alksdjflajsdlkfjalksjflkadjsf.com&tlds=&idnLanguage=&search=search&csrf_pb=3e031623f4516ff491faa3eac63fdf5a) to break this test.

Let's use something like:
```java
String badHost = UUID.randomUUID() + ".palantir.com";
```

Reply:
fixed

Reply:
I think that only went into one of the two tests using this pattern, we need to update both